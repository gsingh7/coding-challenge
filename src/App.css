.App {
  text-align: left;
  margin-left: 20px;
}

.inputTextBox{
  width: 80%;
  padding: 12px 20px;
  margin: 8px 0;
  box-sizing: border-box;
}

.GeoNamesTable{
  margin-top: 20px;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}
