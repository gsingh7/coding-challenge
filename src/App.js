import './App.css';
import DisplayGeoNames from "./components/DisplayGeoNames";

function App() {
  return (
    <div className="App">
        <h1>Virtual Trips Coding Challenge</h1>
        <DisplayGeoNames/>
    </div>
  );
}

export default App;
