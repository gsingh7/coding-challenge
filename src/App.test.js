import { render, screen } from '@testing-library/react';
import App from './App';

test('renders element with latitude', () => {
  render(<App />);
  const linkElement = screen.getByText(/latitude/i);
  expect(linkElement).toBeInTheDocument();
});
