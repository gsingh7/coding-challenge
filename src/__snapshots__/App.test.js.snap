// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayGeoNames renders correctly 1`] = `
<div>
  <input
    className="input-text-box"
    onChange={[Function]}
    placeholder="search box"
    value=""
  />
  <button
    className="toggle-button"
    onClick={[Function]}
  >
    <span>
      Closest Name Match 
      <span>
        is OFF (Toggle ON)
      </span>
    </span>
  </button>
  <div />
</div>
`;
